src/Translation/German/translation.json

```diff
@@ -91,22 +91,19 @@
   "team": {
     "Conrad": {
       "name": "Conrad",
-      "description": "Ex PPT- und XLS-Ingenieur, jetzt CEO von Portato. Sportenthusiast mit grossem Netzwerk.",
-      "image": "src/Assets/Images/team/conrad.jpg"
```

Comment:
Nice catch, removing these unnecessary image lines - thank you!

Reply:
Guess I was the one putting them there in the first place 😅

Reply:
Oh really? Well, wasn't a bad idea - just a different approach now ^^